examples/tv-app/tv-common/tv-app.matter

```diff
@@ -2340,6 +2343,9 @@ endpoint 1 {
     ram      attribute clusterRevision default = 1;
   }
 
+  server cluster MediaPlayback {
```

Comment:
why does this empty cluster exist?

Reply:
In another place in this file we have:

```
server cluster MediaPlayback {
    ram      attribute currentState;
    ram      attribute startTime default = 0xFF;
    ram      attribute duration;
    callback attribute sampledPosition;
    ram      attribute playbackSpeed;
    ram      attribute seekRangeEnd;
    ram      attribute seekRangeStart;
    ram      attribute featureMap default = 0x0003;
    ram      attribute clusterRevision default = 1;
  }
  ```
  
  So wondering here why these are missing. At least featureMap and clusterRevision should probably exist.

Reply:
Hey @andy31415 I've created a PR that fixes this. I've forgot to enable attributes & commands for the given endpoint

https://github.com/project-chip/connectedhomeip/pull/18762